### PR TITLE
test

### PR DESCRIPTION
**Pull request by plane83**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,7 @@ Core Genome Utilities
 A KBase module that exposes core KBase genome and feature utility methods, such as 
 running Blast against a genome.
 
+The name of this module should be changed. 'Core Genome' may refers to something oppsite to 'Pan Genome'. Maybe change the name of this module to 'Genome Core Utilities' or 'Genome basic utilities' will be better for people to understand.
```
Comment (msneddon) [inline README.md]: we could even just change it to genome_util?  We should do that before we migrate to the kbase github namespace.  I can change the name of the repo as well.

**Comment by plane83** [comment]
Yes. Please change it to genome_util.

File: README.md (hunk 1 above)
Comment (plane83) [inline README.md]: Yes. Please change it to genome_util.

**Comment by msneddon** [comment]
ok done.
https://github.com/kbaseIncubator/genome_util

**Pull request merged**